src/models/admin.jsx

```diff
@@ -42,6 +43,10 @@ const approveUser = async (userId) => {
   await updateUser(userId, { status: USER_STATUS.APPROVED });
 };
 
+const deleteUser = async (userId) => {
+  await deleteDoc(doc(database, "Users", userId));
```

Comment:
this would be more complicated
1. for client user, delete the job saves,  see this for how this is stored https://console.firebase.google.com/u/1/project/refugee-one-368a9/firestore/data/~2FUsers~2Fvg3Pq22voBNwNPJw4Qv9gbNMOmJ2
2. for employer, how are we going to do with the jobs he create?
3. for admin, do we need a protection? (this can be discussed later or not do it)

Reply:
Ah that makes sense. I'm not familiar with how to access the usersbjobs in 2) and since we also have some discussion with 3) to do I removed line 47 and added a todo, so it's just an empty function and the logic can be filled in later.